### PR TITLE
Ch setup database

### PR DESCRIPTION
**Pull request by BenoitIT**
### What does this PR do?
This has functions and methods to set up connection  [ecommerce-app-legends-bn](https://github.com/atlp-rwanda/ecommerce-app-legends-bn) App with postgres database during development phase.

#### what are the nature of PL contents?
---config folder which has js file with db connection configurations
---models folder which has User model created for functionality testing as well index.js which has db connection with 
     models
---migration folder which store table structures to be migrated to the database
---scripts to run both create migration events as well as make: db migrate
---connection folder which has folder to notify the the status of connection from the node js console

N.B : migrations and models folder are still in progress as new and more migrations and modls will b created

### Description of Tasks completed
----database connection set-up successfully
----first model created successfully
----first migration created and migrated successfully
--- make model and create migration as well as migrate scripts added and tested
### how to test this?
clone repo , branch git clone https://github.com/atlp-rwanda/ecommerce-app-legends-bn.git
-`run npm i`
-have `postgres database` installed 
-create a database with `ecommerce_db`as its name
-create .env file
-under .env file initialize:`DEV_DATABASE_URL=postgres://<username>:<password>@127.0.0.1:5432/ecommerce_db`
###### remember to take away angled bracket  beside username and password as well colon between those two from connection string above
-start the app and check connectivity in development `npm run dev `
-  run `npm run make:migration`  in terminal to make migation  to create new migration
-rename created migration file with your desired file
- run`npm run migrate` to create new table in database
-  run`npm run generate:seed` to create new database seed 
- run`npm run db:seed` to seed the database
-  run`npm run migrate:undo` to revert migrations
Screen Shot 2023-03-02 at 10 57 22
 database seed

### CODE REVIEW

**Review by Aimeana100 — APPROVED** [review]

**Review by ngoma3 — APPROVED** [review]

**Review by Aimeana100 — CHANGES_REQUESTED** [review]
Sorry 😔, I tested the codes locally, but not noticed that I had conflicts. @BenoitIT  still have to resolve the issues.

File: src/docs/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import swagger from 'swagger-ui-express';
+import home from './home';
+import schemas from './shema';
+
+const { Router } = require('express');
```
Comment (mansurissa) [inline src/docs/index.js]: here use es6 please

**Review by mansurissa — CHANGES_REQUESTED** [review]
Resolve conflicts and work on the comments

**Review by FabCode67 — APPROVED** [review]

**Review by mansurissa — APPROVED** [review]
Looks good now

**Review by Aimeana100 — APPROVED** [review]
Okay, now. It works as expected.

**Comment by FabCode67** [comment]
you need to update readme and remove unwanted schemas

**Review by FabCode67 — APPROVED** [review]

**Review by ngoma3 — APPROVED** [review]

**Review by Kevin-ishimwe — APPROVED** [review]
great work !

**Review by mansurissa — APPROVED** [review]

**Pull request merged**